src/libkomwm.py

```diff
@@ -920,6 +934,7 @@ def cmprepl(a, b):
         return -1
     viskeys.sort(key=functools.cmp_to_key(cmprepl))
 
+    # TODO: Introduce new function to dump `visibility.txt` and `classificator.txt` for better testability
```

Comment:
maybe `visibility.txt` is not needed at all... it seems to be used to cache visibility values so that its faster to load by core, but maybe its as fast to get them from protobuf style files which are loaded anyway;
`classificator.txt` is also used to speed up loading of types hierarchy, but maybe it could be derived from types names as fast..